src/components/MainSidebar/__tests__/__snapshots__/TimeDimensions.spec.js.snap

```diff
@@ -1645,7 +1645,7 @@ exports[`stage missing 1`] = `
           <span
             class="primary"
           >
-            le enrollment date
+            Tracking date
```

Comment:
Something changed here - it is not using the custom name.

Reply:
Well spotted. This code is outdated now, but there was indeed an issue here.